Convert role into collection

l00ptr opened the pull request:
We move the role files under the roles folder and we also adapt the
molecule default scenario and the documentation. It's the first step before adding some plugins to manipulate the config through the LemonLDAP::NG API.

l00ptr committed:
Convert role into collection

We move the role files under the roles folder and we also adapt the
molecule default scenario

l00ptr committed:
Adapt yamlint configuration

l00ptr committed:
Add step to build the collection release in CI

l00ptr committed:
Add galaxy info for the newly created collection

l00ptr committed:
Fix molecule configuration for defautl scenario

l00ptr committed:
Fix namespace inside molecule converge playbook

maxbes commented on galaxy.yml:
```diff
@@ -0,0 +1,67 @@
+version: 0.0.1
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
```
Please update the README to reflect the new role path in the example

l00ptr committed:
Adapt documentation for Ansibe collection

l00ptr committed:
Make molecule scenario run locally & in CI

We remove .github action specific stuff to run our molecule scenario
in CI with different distrib and let molecule do the job

l00ptr committed:
Fix linting error

l00ptr committed:
Replace deprecated molecule[docker] by plugins

l00ptr committed:
Enable powertool on RedHat clone

l00ptr committed:
Install lemonldap-ng-doc instead of missing fr

l00ptr commented:
Any feedback for this (long standing) MR ?

maxbes commented:
@coudot looks good to me

l00ptr commented:
I will have some free time to work on some improvement / maintenance on this collection the 2 next weeks. I will create a new branch starting from this unmerged one.

coudot commented:
Hello @l00ptr 

following review from @maxbes I merged this PR. You can start a new one from master branch if you want.

Thanks again for your contribution.

l00ptr commented:
@coudot great thx!
